src/sentry/conf/server.py

```diff
@@ -1422,14 +1422,16 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:session-replay-sdk-errors-only": False,
     # Enable data scrubbing of replay recording payloads in Relay.
     "organizations:session-replay-recording-scrubbing": False,
-    # Enables subquery optimizations for the replay_index page
+    # Enable subquery optimizations for the replay_index page
     "organizations:session-replay-index-subquery": False,
     # Enable the new suggested assignees feature
     "organizations:streamline-targeting-context": False,
     # Enable the new experimental starfish view
     "organizations:starfish-view": False,
     # Enable starfish endpoint that's used for regressing testing purposes
     "organizations:starfish-test-endpoint": False,
+    # Replace sentry logos with pride logos
```

Comment:
```suggestion
    # Replace the footer sentry logo with a Sentry pride logo
```

Reply:
do we only want to replace the ones we display in the footer?

Reply:
Yep, just this one for now.